.cirrus.yml

```diff
@@ -708,6 +703,30 @@ podman_machine_task:
     always: *int_logs_artifacts
 
 
+podman_machine_aarch64_task:
+    name: *std_name_fmt
+    alias: podman_machine_aarch64
+    only_if: *not_tag_branch_build_docs
+    depends_on:
+        - build
+        - local_integration_test
+        - remote_integration_test
+        - container_integration_test
+        - rootless_integration_test
+    ec2_instance:
+        <<: *standard_build_ec2_aarch64
+        type: c6g.metal  # Bare-metal instance is required
```

Comment:
Non-blocking: This will break in `hack/get_ci_vm.sh`,  If you need/want to make it work, define this type via an env. var. `EC2_INST_TYPE`.  [See this PR for examples.](https://github.com/containers/podman/pull/15145/files)

Reply:
maybe this is best documented in the get_ci_vm script ?

Reply:
Then I'd have to document it in every repo's copy of the script :disappointed:   The intended way for people to find this out was to run `hack/get_ci_vm.sh` on the task in question and read the helpful error message.  But maybe that was too optimistic on my part :grin: 

We can fix this later, no reason to hold up this PR (unless you need to re-push for some reason).

Reply:
ack, thanks!